docs/quick-start/index.md

```diff
@@ -0,0 +1,487 @@
+# MLServer Quick-Start Guide
```

Comment:
I still think calling it a quick start guide is misleading. It's a tutorial that walks through a specific learning path to achieve a use-case. (See https://diataxis.fr/ for more on definitions).

In contrast, a "quickstart" is something that is designed to get you up and running with the **basics** of the tool in just a few minutes.

Reply:
Fair point. How about calling this section "Getting Started"?